src/fft_cufft.f90

```diff
@@ -408,24 +409,18 @@ subroutine c2c_1m_x(inout, isign, plan1)
       integer, intent(IN) :: isign
       integer*4, intent(IN) :: plan1
 
-      complex(mytype), dimension(:, :, :), allocatable :: output
       integer :: istat
 
-      allocate (output, mold=inout)
 #ifdef DOUBLE_PREC
-      !$acc host_data use_device(inout,output)
-      istat = cufftExecZ2Z(plan1, inout, output, isign)
+      !$acc host_data use_device(inout)
+      istat = cufftExecZ2Z(plan1, inout, inout, isign)
```

Comment:
Likewise - is `inout` a problem/reserved keyword?

Reply:
inout is a bit everywhere and comes from first implementation? For the moment I’ll leave it there and have a PR where we change it consistently

Reply:
ok - sounds good, and compiler isn't complaining